runtime/centrifuge/src/migrations.rs

```diff
@@ -14,12 +14,22 @@ use crate::Runtime;
 
 /// The migration set for Centrifuge @ Polkadot.
 /// It includes all the migrations that have to be applied on that chain.
-pub type UpgradeCentrifuge1401 = (
+pub type UpgradeCentrifuge1402 = (
 	// Clear OutboundMessageNonceStore
 	frame_support::migrations::VersionedMigration<
 		0,
 		1,
-		runtime_common::migrations::liquidity_pools_gateway::Migration<Runtime>,
+		runtime_common::migrations::liquidity_pools_gateway::clear_outbound_nonce::Migration<
+			Runtime,
+		>,
+		pallet_liquidity_pools_gateway::Pallet<Runtime>,
+		<Runtime as frame_system::Config>::DbWeight,
+	>,
+	// Remove deprecated DomainRouters entries
+	frame_support::migrations::VersionedMigration<
+		1,
+		2,
+		runtime_common::migrations::liquidity_pools_gateway::clear_deprecated_domain_router_entries::Migration<Runtime>,
 		pallet_liquidity_pools_gateway::Pallet<Runtime>,
 		<Runtime as frame_system::Config>::DbWeight,
 	>,
```

Comment:
FYI: The issue with `try-runtime` decoding errors were the idempocy checks which re-executed the pre- and post-checks. It appears to me that during re-execution, the values were already migrated such that they failed to decode for `v0`. Therefore, I bumped the storage version of the gateway once again and wrapped the migration into a `VersionedMigration`.

Reply:
> I bumped the storage version of the gateway once again

That's weird. From 0.14.0, the version was already increment, isn't?